src/main/java/org/graylog2/log4j2/GelfAppender.java

```diff
@@ -41,7 +41,7 @@
 @Plugin(name = "GELF", category = "Core", elementType = "appender", printObject = true)
 public class GelfAppender extends AbstractAppender {
     private static final long serialVersionUID = 4796033328540158817L;
-
+    private static final String REGEX_IP_ADDRESS = "\\d+(\\.\\d+){3}";
```

Comment:
This will break with IPv6 address literals.

Better check with something like [`InetAddresses#isInetAddress(java.lang.String)`](https://google.github.io/guava/releases/19.0/api/docs/com/google/common/net/InetAddresses.html#isInetAddress(java.lang.String)) (disclaimer: Guava is not used and this project and shouldn't be pulled in just for this method).